includes/modules/pages/contact_us/header_php.php

```diff
@@ -26,57 +26,65 @@
 
   $zc_validate_email = zen_validate_email($email_address);
 
-  if ($zc_validate_email and !empty($enquiry) and !empty($name) && $error == FALSE) {
+  if (($zc_validate_email && !empty($enquiry) && !empty($name)) && $error == FALSE) {

```

Comment:
```suggestion
  if ($zc_validate_email && !empty($enquiry) && !empty($name) && $error == FALSE) {
```

Thanks for considering the parentheses which denote the prior meaning; however in this case they are unnecessary.